### PR TITLE
[update]:responsive and readme update

### PR DESCRIPTION
**Pull request by ultronjs**
-Added Responsiveness
-ReadMe File Update
-Updated title and Favicon

### CODE REVIEW

**Review by mariyasada — APPROVED** [review]

File: app.css [hunk 1]
```diff
@@ -204,7 +215,82 @@ body{
...
+    gap: .5rem;
+    grid-template-areas:
+        "cart_price_container"
+        "cart_card_container"
+        
+    }
+    .cart_card_container{
+        grid-area: cart_card_container;
+    }
+
```
Comment (ankit9015) [inline app.css]: You can erase unused line spaces

**Review by ankit9015 — APPROVED** [review]
You can merge it, everything looks fine,

File: app.css (hunk 1 above)
Comment (ultronjs) [reply app.css]: Done

**Comment by ultronjs** [comment]
Merging after 2 approval

**Pull request merged**